domino/_version.py

```diff
@@ -1 +1 @@
-__version__ = '1.0.4'
+__version__ = '1.0.5'
```

Comment:
A discussion with Med in a Slack thread came up re: changing the version scheme of this package to match the corresponding Domino version. A user would know immediately, for example, that to access a API features associated Domino 4.5.0, they would need to pip install `python-domino==4.5.0`.

@mpkouznetsov @dmitriyvolk -- any thoughts?

Reply:
No strong opinion. python-domino supports multiple versions of domino so aligning the versioning schemes will sort of hint that we support _only_ the latest version. Still, it probably makes more sense to align.

Reply:
Discussed with Pablo and we decided not to change versioning scheme at this moment.